shenyu-common/pom.xml

```diff
@@ -63,5 +64,10 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-collections4</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>com.github.ben-manes.caffeine</groupId>
+            <artifactId>caffeine</artifactId>
```

Comment:
add in license file？

Reply:
Of course, it's done, using version 2.9.3 from SpringBoot 2.6.8.